stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ChatApp.kt

```diff
@@ -23,16 +28,23 @@ class ChatApp : Application() {
         dateFormatter = DateFormatter.from(this)
 
         setupStreamSdk()
+        initializeToggleService()
         connectUser()
     }
 
+    @OptIn(InternalStreamChatApi::class)
+    private fun initializeToggleService() {
+        ToggleService.init(applicationContext, mapOf(ToggleService.TOGGLE_KEY_OFFLINE to BuildConfig.DEBUG))
+    }
+
+    @OptIn(ExperimentalStreamChatApi::class)
     private fun setupStreamSdk() {
-        val client = ChatClient.Builder("qx5us2v6xvmh", applicationContext)
+        val offlinePlugin = OfflinePlugin(Config(userPresence = true, persistenceEnabled = true))
```

Comment:
We need to use `OfflinePlugin` for debug build in compose sample to catch all issues before initial release of domain merge